src/wp-includes/rest-api/endpoints/class-wp-rest-autosaves-controller.php

```diff
@@ -494,4 +498,32 @@ public function get_collection_params() {
 			'context' => $this->get_context_param( array( 'default' => 'view' ) ),
 		);
 	}
+
+	/**
+	 * Gets the post, if the ID is valid.
+	 *
+	 * @since 6.7.0
+	 *
+	 * @param int $id Supplied ID.
+	 * @return WP_Post|WP_Error Post object if ID is valid, WP_Error otherwise.
+	 */
+	protected function get_post( $id ) {
```

Comment:
Could we not use $this->parent_controller->get_post, if we make that method public?

Reply:
Why would changing the visablity to public break backwards compat?

Reply:
My thinking is that the method signatures in both the parent class and the child class should match.
If a child class extends a parent class and overrides one of its methods, changing the visibility to `public` in the parent class will cause the signatures to no longer match, and PHP will throw a fatal error.

Here’s a code snippet that illustrates my concern: https://3v4l.org/XW2Zj